gui/design.lua

```diff
@@ -1565,6 +1567,94 @@ function DesignScreen:onDismiss()
     view = nil
 end
 
+
+-- --------------------------------
+-- DesignToolbarOverlay
+--
+
+DesignToolbarOverlay = defclass(DesignToolbarOverlay, overlay.OverlayWidget)
+DesignToolbarOverlay.ATTRS{
+    desc='Adds a button to the toolbar at the bottom of the screen for launching gui/design.',
+    default_pos={x=50, y=-1},
```

Comment:
This position isn't going to work:
![image](https://github.com/user-attachments/assets/0237d13f-8d38-4a73-96dd-205fbc497212)

mayyybe here in the dig toolbar?
![image](https://github.com/user-attachments/assets/ff452e78-29c1-4331-a4c4-c5e66aa2e795)

though that bar is already quite crowded..